src/agent/src/config.rs

```diff
@@ -25,6 +25,7 @@ const LOG_VPORT_OPTION: &str = "agent.log_vport";
 const CONTAINER_PIPE_SIZE_OPTION: &str = "agent.container_pipe_size";
 const UNIFIED_CGROUP_HIERARCHY_OPTION: &str = "agent.unified_cgroup_hierarchy";
 const CONFIG_FILE: &str = "agent.config_file";
+const REST_API_OPTION: &str = "agent.rest_api";
```

Comment:
Hopefully we can move this to a config file before too long. Fine to configure via the cmdline for now.